lib/easyadmob/easy_admob_bannerAd.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:google_mobile_ads/google_mobile_ads.dart';
+
+class EasyAdmobBannerAd {
```

Comment:
use _ where the word is capital in class name. in this case, it would be easy_admob_banner_ad